sanitize.go

```diff
@@ -161,6 +175,19 @@ func (s Sanitizer) sanitizeRec(v reflect.Value) error {
 		field := v.Field(i)
 		fkind := field.Kind()
 
+		// Prioritize custom sanitizers; tag's value can be re-resolved inside the sanitizer
+		tags := s.fieldTags(v.Type().Field(i).Tag)
+		for tag, _ := range tags {
+			sanitizerFunc, ok := s.sanitizersByName[tag]
+			if !ok {
```

Comment:
(Just a comment)
I guess this is ok, since the default behaviour of the library is to ignore unknown tags. But now I wonder if we are doing the right thing by not even logging it as a potential problem. I'll give it more thought.

Reply:
That makes sense. As it currently stands, the built-in tags exist in each type-sanitizer, so it's not possible for the custom sanitizer to know whether a tag directive is a custom sanitizer, a built-in sanitizer, or does not exist.

This could be fixed with a larger-scale overhaul of the type sanitizers by registering them in a similar fashion to the custom sanitizers, if that's a direction you'd like to move to over time.

Reply:
I think this way is good enough. I think it's better to keep the behaviour consistent for both custom and standard sanitizers.